src/subscription.ts

```
@@ -0,0 +1,244 @@
+import { Request, Response } from 'express';
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+import Stripe from 'stripe';
+import { Request, Response } from 'express';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Request'. Consider to delete it or rename it.

'Request' is defined here:


(lint/suspicious/noRedeclare)